test/common/helpers.test.js

```diff
@@ -222,3 +222,85 @@ test('getHashesMap should throw error if the checksums array contains duplicate
         'Should have throw error but didnt!'
     );
 });
+
+/**
+ * Test the convertToMarthaV3Response() function
```

Comment:
In addition to these unit tests, what is your thought about adding a test to the overall marthav3 test where something in the DRS response should come back as null in the overall response?

Reply:
The way mocking is set up currently, it is tricky to resolve some fields in the response to null. I will take a look at it to see if it can be done

Reply:
I was able to add such a test by changing the sandbox setup for only this test

Reply:
Thanks for adding that mocked end-to-end test!